web/src/components/ContactItem.tsx

```diff
@@ -20,17 +22,29 @@ const StyledLink = styled(Link)`
   ${helpers.adaptiveFontSize};
 `
 
+
+const StyledExternalLinkIcon = styled.img`
+  width: 16px;
+  height: 16px;
+  flex-shrink: 0;
+  object-fit: contain;
+  margin-left: 8px; // Добавляем отступ между текстом и иконкой
+`
+
 type ContactItemProps = {
   iconSrc: string
   iconAlt: string
   link: string
   content: string
+  isExternalLink?: boolean 
 }
 
-const ContactItem = ({ iconSrc, iconAlt, link, content }: ContactItemProps): ReactElement => (
+const ContactItem = ({ iconSrc, iconAlt, link, content, isExternalLink }: ContactItemProps): ReactElement => (
```

Comment:
```suggestion
const ContactItem = ({ iconSrc, iconAlt, link, content, isExternalLink = false }: ContactItemProps): ReactElement => (
```
🔧  Just to add a default false; it's not really necessary but makes it clearer for people reading the code in the future that the usual case is to not use the external link